sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/mdesupport/MdeSupportBridgeHelpers.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.encryption.mdesupport;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MdeSupportBridgeHelpers {
```

Comment:
Shouldn't we add `@Warning` annotation to this class and all its public methods ?

Reply:
Done, btw in cosmos core project we dont have warning for BridgeHelpers, something we should do there as well